dynatrace/sum.go

```diff
@@ -0,0 +1,21 @@
+package dynatrace
+
+import (
+	"github.com/dynatrace-oss/dynatrace-metric-utils-go/metric"
+	"go.opentelemetry.io/otel/metric/number"
+	"go.opentelemetry.io/otel/sdk/export/metric/aggregation"
+)
+
+func valueOptForSum(sum aggregation.Sum, monotonic bool, kind number.Kind) (metric.MetricOption, error) {
+	// export histogram
+	value, err := sum.Sum()
+	if err != nil {
+		return nil, err
+	}
+
+	if monotonic {
+		return metric.WithFloatCounterValueDelta(value.CoerceToFloat64(kind)), nil
```

Comment:
this currently does not work for async counters right?

Reply:
See the updated example for an.... example :)